internal/bootstrap/bootstrap_config_test.go

```diff
@@ -91,19 +94,6 @@ const (
     ]
   }
```

Comment:
just checking, is this file diff bigger because the changes weren't synced recently?

Reply:
It's because the original changes from #90 weren't synced via the Makefile target, and were a bit different from the Consul changes

Now things are the same on both sides